UnitsNet/QuantityValue.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright © 2007 Andreas Gullberg Larsen (angularsen@gmail.com).
```

Comment:
Wrong year

Reply:
Fixed in master and in branch, I saw most headers were out of...date... 😎